plugins/inlinetoolbar/plugin.js

```diff
@@ -99,6 +135,62 @@
 					that.attach( element, false );
 				} ) );
 			};
+
+			/**
+			 * Render items and attach view to DOM element.
+			 *
+			 * @param {CKEDITOR.dom.element} element The element to which the panel is attached.
+			 */
+			CKEDITOR.ui.inlineToolbar.prototype.create = function( element ) {
+				this._view.renderItems( this._items );
+				this._view.create( element );
+			};
+
+			/**
+			 * Adds an item to the inline toolbar.
+			 *
+			 * @param {String} name The menu item name.
+			 * @param {Object} element instance of {CKEDITOR.ui.button/CKEDITOR.ui.richCombo}
```

Comment:
This is not how you mark param type. See other code docs, JSDuck wiki for reference.